src/chat.rs

```diff
@@ -163,16 +298,17 @@ impl Chat {
                 .sql
                 .query_get_value(
                     context,
-                    "SELECT c.addr FROM chats_contacts cc  \
-                     LEFT JOIN contacts c ON c.id=cc.contact_id  \
-                     WHERE cc.chat_id=?;",
-                    params![self.id as i32],
+                    "SELECT c.addr
```

Comment:
I don't really like the idea of removing `\` because it prevents `cargo fmt` from changing the amount of leading whitespace. (Not a change request)

Reply:
It was what was concluded in https://github.com/deltachat/deltachat-core-rust/pull/852 as the preferred style to format SQL for it's safety.  That discussion didn't really consider rust-fmt considerations, it is true that if the indentation changes the block looks better if it's manually reformatted.  I also didn't know that cargo-fmt re-indented things if they had a `\` at the end off the line, but it does explain the behaviour of ignored leading whitespace.

I think if you feel like discussing this more we should re-open https://github.com/deltachat/deltachat-core-rust/pull/852 and carry on there.